utils/test/src/main/kotlin/Utils.kt

```diff
@@ -80,7 +80,9 @@ fun patchExpectedResult(
         .replaceIfNotNull("<REPLACE_URL_PROCESSED>", urlProcessed)
 }
 
-class ReplacementsBuilder {
+class ReplacementsBuilder(
+    val urlProcessed: String
```

Comment:
It seems a bit arbitrary to make `urlProcessed` available here just because `CarthageFunTest` happens to require it. Can't we simply remove `<REPLACE_GITHUB_PROJECT>` and hard-code it to "oss-review-toolkit" in the YAML files, similar to like you did with some other patterns? This would then also work with the `buildMap`-based approach I'm proposing,

Reply:
This is not just `CarthageFunTest` but also `GoDepFunTest` which I haven't migrated yet.

Anyhow, I'll just turn it into your build map approach and drop the `<REPLACE_GITHUB_PROJECT>` patterns.